src/controllers/notification.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable require-jsdoc */
+import { newTripNotification } from '../utils/emailHelper';
+import Models from '../database/models';
+import { onlineClients } from '../utils/socket';
+import setLanguage from '../utils/international';
+
+class notificationsController {
+  static async sendNotification(user, type, request, host) {
```

Comment:
@RusimbiP Would you endeavor to document ```sendNotification``` method like so

Reply:
Sure. Thanks @hakoemmy

Reply:
@RusimbiP Good to know